wikidict/svg.py

```diff
@@ -25,7 +25,7 @@ def get(formula: str) -> str:
 
 
 def set(formula: str, svg_raw: str) -> None:
-    print(f"{formula!r}: {svg_raw!r},")
+    # print(f"{formula!r}: {svg_raw!r},")
```

Comment:
Is there an issue?

Reply:
it was a bit verbose when I was debugging. I will put it back.

Reply:
If that prints something, then we are missing some SVGs in our cache. Just copy-paste the lines, and run `python sort-svg-cache.py` (+ commit the changes).

Reply:
mmm, now I need to remember when it printed :)